### PR TITLE
Removed Google+

### PR DESCRIPTION
**Pull request by pe7er**
PR for issue https://github.com/joomla/joomla-websites/issues/1311

### CODE REVIEW

**Comment by brianteeman** [comment]
Best me to it :)

File: joomlarrssb.xml [hunk 1]
```diff
@@ -25,7 +25,6 @@
 					<option value="0">JNO</option>
 					<option value="1">JYES</option>
 				</field>
-				<field name="displayGoogle" type="radio" class="btn-group btn-group-yesno" default="1" label="PLG_CONTENT_JOOMLARRSSB_FIELDSET_DISPLAY_GOOGLE_LABEL" description="PLG_CONTENT_JOOMLARRSSB_FIELDSET_DISPLAY_GOOGLE_DESCRIPTION">
```
Comment (brianteeman) [inline joomlarrssb.xml:28]: You need to remove the next two lines <option...

**Review by brianteeman — CHANGES_REQUESTED** [review]

File: joomlarrssb.xml (hunk 1 above)
Comment (pe7er) [reply joomlarrssb.xml:28]: Thanks Brian!

**Comment by pe7er** [comment]
Fixed error in XML manifest. Thanks @brianteeman !

**Pull request merged**